R/sass.R

```diff
@@ -301,6 +301,8 @@ sass_partial <- function(
 #'   directory name.
 #' @param fileext the output file extension. The default is `".min.css"` for
 #'   compressed and compact output styles; otherwise, its `".css"`.
+#' @param temp_dir the output directory to write the temporary file to. The
```

Comment:
Let's do `tmpdir` instead of `temp_dir` (normally I prefer the former, but we've already decided to use the same naming as `tempfile()`)